source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,85 @@
+
+Playbook roles
+---------------
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who are added to a playbook.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility as well as functional settings. They do not have access to the System Console and their privileges are managed by the System Admin. Members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook.
+
+.. note::
+
+Before you make changes to permissions and to ensure that you don't lose access to playbooks, . To do this, navigate to the playbook you're a member of. Select the **Manage Access** icon and change your role from **Member** to **Admin**.
+
+Playbooks permissions
+---------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view public and private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs. They also can't create public or private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+You can also set permissions for read-only playbooks that do allow members to create new public or private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Then, check **Create Public Playbook** and **Create Private Playbook**.
+4. Select **Save**.
+
+I want to manage how playbooks are accessed
+-------------------------------------------
+
+A public playbook and its associated runs can be viewed by any member of a team. Playbooks can be converted from public to private, but once a playbook is private it can't be converted back to public.
+
+I want to restrict who can convert playbooks from public to private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can control whether Members can convert playbooks from public to private.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from public to private:
```

Comment:
This reads like it's a necessary next step for admins, but if they do the above, admins should inherit this by default. Maybe rephrase as,
```suggestion
Alternatively, to restrict this action so only Playbook Admins can convert playbooks from public to private, uncheck the setting above and:
```